ynput/ayon-frontend
# Browser: Unify entity details panel

## PR Checklist



-   [x] This comment contains a description of changes
-   [x] Referenced issue is linked

## Description of changes

- Replaces the old browser details panel with the new one use on dashboard.
- You can now use the activity feed with folders.
- New Representations tab for versions.

![image](https://github.com/ynput/ayon-frontend/assets/49156310/35acd2dc-8afb-4990-ad94-39df62fe4fbf)


https://github.com/ynput/ayon-frontend/assets/49156310/0eea987f-bf9e-4b69-a1dd-b0c08a89aee6


### Technical details

Removes a lot of duplicate code and unifies a details panel for entities into one component and design.

### Testing

1. Open a project
2. Select any entity, (folder, task, version).
3. Click around in the details panel.

1. Select a version
2. Select "representations" tab
3. Select a representation from the table
4. The slide out should appear.
